wordpress-mobile/WordPress-iOS
# Remove helpshift credentials

Ref #9020 

To test:

- Obtain updated credentials files from [this P2](https://wp.me/P77Llu-8m1-p2).
- Update your local `wpcom_app_credentials`.
- Build the project.
- Verify Derived Sources > ApiCredentials.m no longer has Helpshift entries.
